### PR TITLE
Switch from kotlin mockito to mockk

### PR DESCRIPTION
**Pull request by DanielJette**
### What does this change accomplish?

1. Remove kotlin mockito dependencies
2. Add dependency on mockk
3. Update tests to use mockk
4. Convert `DeviceIdentifier` into top-level functions

### How have you achieved it?

Mockito is not able to mock top-level functions and I'm planning to also convert `ScreenshotUtility` and `OutputFileUtility` to top-level functions in a future PR. So, I took the opportunity to swap out Mockito for mockk in this PR.

### Tophat instructions

- All tests should pass

### CODE REVIEW

File: Library/src/test/java/dev/testify/RegionCompareTest.kt [hunk 1]
```diff
@@ -93,7 +89,14 @@ class RegionCompareTest {
...
         val baselineBitmap = mockBitmap(0xff0000ff.toInt())
 
+        // TODO:
         val rect = mockRect(6, 5, 10, 10)
+//            mockk<Rect>(relaxed = true) {
+//            every { this@mockk.contains(any(), any()) } answers {
+//
+//                true
+//            }
+//        }//mockRect(6, 5, 10, 10)
```
[the remark this answers is not in the document]
Comment (DanielJette) [reply Library/src/test/java/dev/testify/RegionCompareTest.kt:99]: 😠 Yes. Will fix. Thanks for spotting it

File: Library/build.gradle [hunk 2]
```diff
@@ -66,9 +66,9 @@ android {
             exclude group: 'org.jetbrains.kotlin', module: 'kotlin-stdlib-common'
         }
 
-        testImplementation "org.mockito:mockito-core:${versions.mockito2}"
-        testImplementation "org.mockito.kotlin:mockito-kotlin:${versions.mockitokotlin}"
+        testImplementation "io.mockk:mockk:${versions.mockk}"
         testImplementation "org.jetbrains.kotlinx:kotlinx-coroutines-test:${versions.kotlinx}"
+        testImplementation "org.slf4j:slf4j-jdk14:${versions.slf4j}"
```
Comment (jaredh) [inline Library/build.gradle:71]: I was a bit 🤨 at the JDK14 reference, but checking their artifacts, it's apparently for JDK _1.4_ not _14_. TIL.

**Review by jaredh — APPROVED** [review]
LGTM.

Nice feature set on MockK.

File: Library/build.gradle (hunk 2 above)
Comment (DanielJette) [reply Library/build.gradle:71]: Yeah, I was thrown off a bit by this.

**Pull request merged**